src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2917,6 +2917,51 @@ private CorInfoTypeWithMod getArgType(CORINFO_SIG_INFO* sig, CORINFO_ARG_LIST_ST
             }
         }
 
+        private int getExactClasses(CORINFO_CLASS_STRUCT_* baseType, int maxExactClasses, CORINFO_CLASS_STRUCT_** exactClsRet)
+        {
+#if !READYTORUN
```

Comment:
We have CorInfoImpl.ReadyToRun.cs (R2R) and CorInfoImpl.RyuJit.cs ([sic], NativeAOT) that has unshareable implementations like this - if the whole body is under ifdef, it's better to not put it in the shared file.